modules/restore-replace-stopped-baremetal-etcd-member.adoc

```diff
@@ -0,0 +1,481 @@
+= Replacing an unhealthy bare metal etcd member whose machine is not running or whose node is not ready
+
+This procedure details the steps to replace a bare metal etcd member that is unhealthy either because the machine is not running or because the node is not ready.
+
+.Prerequisites
+
+* You have identified the unhealthy bare metal etcd member.
+* You have verified that either the machine is not running or the node is not ready.
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have taken an etcd backup.
++
+[IMPORTANT]
+====
+It is important to take an etcd backup before performing this procedure so that your cluster can be restored if you encounter any issues.
+====
+
+.Procedure
+
+. Remove the unhealthy member.
+
+.. Choose a pod that is _not_ on the affected node:
++
+In a terminal that has access to the cluster as a `cluster-admin` user, run the following command:
++
+[source,terminal]
+----
+$ oc get pods -n openshift-etcd | grep -v etcd-quorum-guard | grep etcd
+----
++
+.Example output
+[source,terminal]
+----
+etcd-ip-10-0-131-183.ec2.internal                3/3     Running     0          123m
+etcd-ip-10-0-164-97.ec2.internal                 3/3     Running     0          123m
+etcd-ip-10-0-154-204.ec2.internal                3/3     Running     0          124m
+----
+
+.. Connect to the running etcd container, passing in the name of a pod that is not on the affected node:
++
+In a terminal that has access to the cluster as a `cluster-admin` user, run the following command:
++
+[source,terminal]
+----
+$ oc rsh -n openshift-etcd etcd-ip-10-0-154-204.ec2.internal
+----
+
+.. View the member list:
++
+[source,terminal]
+----
+sh-4.2# etcdctl member list -w table
+----
++
+.Example output
+[source,terminal]
+----
++------------------+---------+------------------------------+---------------------------+---------------------------+
+|        ID        | STATUS  |             NAME             |        PEER ADDRS         |       CLIENT ADDRS        |
++------------------+---------+------------------------------+---------------------------+---------------------------+
+| 6fc1e7c9db35841d | started | ip-10-0-131-183.ec2.internal | https://10.0.131.183:2380 | https://10.0.131.183:2379 |
+| 757b6793e2408b6c | started |  ip-10-0-164-97.ec2.internal |  https://10.0.164.97:2380 |  https://10.0.164.97:2379 |
+| ca8c2990a0aa29d1 | started | ip-10-0-154-204.ec2.internal | https://10.0.154.204:2380 | https://10.0.154.204:2379 |
++------------------+---------+------------------------------+---------------------------+---------------------------+
+----
++
+Take note of the ID and the name of the unhealthy etcd member, because these values are needed later in the procedure. The `$ etcdctl endpoint health` command will list the removed member until the procedure of replacement is finished and a new member is added.
+
+.. Remove the unhealthy etcd member by providing the ID to the `etcdctl member remove` command:
```

Comment:
This part should be moved to after bmh deletion

Reply:
@lshilin-r It appears the problem is with the sentence "The `$ etcdctl endpoint health` command will list the removed member until the procedure of replacement is finished and a new member is added.", is this correct? If so, move only this sentence to after bmh deletion?

Reply:
@lshilin-r Please forgive me, I've not moved this yet - I need more clarification.

Reply:
No need to move it. I thought I mentioned it here, but may be did it on slack only